lib/shared/snackbar.dart

```diff
@@ -15,55 +15,65 @@ void showSnackbar(
   IconData? trailingIcon,
   void Function()? trailingAction,
 }) {
-  int wordCount = RegExp(r'[\w-]+').allMatches(text).length;
-  SnackBar snackBar = SnackBar(
+  final ThemeData theme = Theme.of(context);
+  final int wordCount = RegExp(r'[\w-]+').allMatches(text).length;
+
+  GetSnackBar snackBar = GetSnackBar(
     duration: duration ?? Duration(milliseconds: max(4000, 1000 * wordCount)), // Assuming 60 WPM or 1 WPS
-    backgroundColor: backgroundColor,
-    content: Row(
-      mainAxisAlignment: MainAxisAlignment.spaceBetween,
-      children: [
-        if (leadingIcon != null)
-          Icon(
+    messageText: Text(
+      text,
+      style: const TextStyle(color: Colors.white),
+    ),
+    icon: leadingIcon != null
+        ? Icon(
             leadingIcon,
             color: leadingIconColor,
-          ),
-        if (leadingIcon != null) const SizedBox(width: 8.0),
-        Expanded(
-          child: Text(
-            text,
-          ),
-        ),
-        if (trailingIcon != null)
-          SizedBox(
+          )
+        : null,
+    backgroundColor: backgroundColor ?? const Color(0xFF303030),

```

Comment:
For the background color [M3 guidelines](https://m3.material.io/components/snackbar/specs#200c7ca0-830a-470e-afb4-4e3e3fef45c9)  recommends `onInverseSurface`

Reply:
I think you mean `inverseSurface` for the background and `onInverseSurface` for the foreground. That looks fine to me, so I updated.

| Light | Dark |
|--------|--------|
| ![image](https://github.com/thunder-app/thunder/assets/7417301/28a37569-2553-42e7-a94d-648026b387ff) | ![image](https://github.com/thunder-app/thunder/assets/7417301/94298520-4952-4020-b829-b04d0344b6f1) |